src/towncrier/test/test_create.py

```diff
@@ -93,6 +98,26 @@ def test_edit_abort(self):
                 self.assertEqual([], os.listdir("foo/newsfragments"))
                 self.assertEqual(1, result.exit_code)
 
+    def test_message(self):
+        """Set fragment message in command line"""
```

Comment:
Maybe add more info about the targeted test case / functionality.

Without documentation, it's hard otherwise to know what is the intended behaviour.

```suggestion
    def test_message(self):
        """
        When creating a new fragment the content can be passed as a
        command line argument.
        The text editor is not invoked.
        """
```